src/features/login/LoginScreen.tsx

```diff
@@ -1,20 +1,48 @@
 import React from 'react';
 import {View} from 'react-native';
+import {connect} from 'react-redux';
 
 import {loginStyles} from './loginScreenStyles'
-import {MyButton} from '../../components/button/ButtonComponent.tsx';
-import {InputText} from '../../components/input/InputComponent.tsx';
+import {MyButton} from '../../components/button/ButtonComponent';
+import {InputText} from '../../components/input/InputComponent';
+import {testData} from '../../services/authService';
 
 
-export const LoginScreen = () => (
-    <View style={loginStyles.container}>
-        <InputText placeholder='Enter your Email'
-        keyboardType= 'email-address'/>
-        <InputText placeholder='Enter your password'
-                   secureTextEntry
-        />
-        <MyButton
-            text='Login'
-        />
-    </View>
-);
+export const LoginScreen = (props) => {
+
+    return(
+        <View style={loginStyles.container}>
+            <InputText placeholder='Enter your Email'
+                        keyboardType= 'email-address'
+                        autoCapitalize = 'none'
+                        value={props.email}
+            />
+            <InputText placeholder='Enter your password'
+                        autoCapitalize = 'none'
+                        secureTextEntry
+                        value={props.password}
+            />
+            <MyButton
+                text='Login'
+                onPress = {props.logIn}
+            />
+        </View>
+    );
+};
+
+const mapDispatchToProps = (dispatch) => {
```

Comment:
min func, this func can return object directly (...) => ({...}) no need to use return keyword.